patchdeps.py

```diff
@@ -172,7 +172,7 @@ def depends_dot(args, patches, depends):
     if args.randomize:
         res += "start=random\n"
 
-    for p in patches:
+    for i, p in enumerate(patches):
         label = dot_escape_string(str(p))
         label = "\\n".join(textwrap.wrap(label, 25))
         res += """{} [label="{}"]\n""".format(p.number, label)
```

Comment:
Commit " refactor: Replace Changeset.number by enumerate() " leaves one reference to `p.number` here (and one more a few lines down). They are removed in a future commit, but those changes should be moved here.